pkg/testutils/modulereleasemeta.go

```diff
@@ -147,3 +153,33 @@ func UpdateAllModuleReleaseMetaChannelVersions(ctx context.Context, client clien
 	}
 	return nil
 }
+
+func ModuleReleaseMetaBetaValueIsCorrect(ctx context.Context, client client.Client, namespace, name string,
+	expectedValue bool,
+) error {
+	meta := &v1beta2.ModuleReleaseMeta{}
+	if err := client.Get(ctx, types.NamespacedName{Namespace: namespace, Name: name}, meta); err != nil {
+		return err
+	}
+
+	if meta.Spec.Beta != expectedValue {
+		return ErrBetaValueNotCorrect
+	}
+
+	return nil
+}
+
+func ModuleReleaseMetaInternalValueIsCorrect(ctx context.Context, client client.Client, namespace, name string,
+	expectedValue bool,
+) error {
+	meta := &v1beta2.ModuleReleaseMeta{}
+	if err := client.Get(ctx, types.NamespacedName{Namespace: namespace, Name: name}, meta); err != nil {
+		return err
+	}
+
+	if meta.Spec.Beta != expectedValue {
```

Comment:
This must be `meta.Spec.Internal`. Btw. We would also have `IsInternal` / `IsBeta` helpers on the MRM